pandasai/constants.py

```diff
@@ -73,3 +74,5 @@
     "vars",
     "zip",
 ]
+PLOT_KEY_WORDS = ["plot", "plt", "graph", "chart"]
+PLT_CODE = "\nplt.show()"
```

Comment:
Should it be `"\nplt.show(blocking = False)"`

Reply:
@yzaparto totally agree!